### ISSUE TITLE
cannot import name 'pyPeriodicTable' from 'rdkit.Chem'

### ISSUE THREAD

**Issue by fdmm1989** [rdkit/rdkit#2741]
Hi,
When I used pychem, I met the following problem:
cannot import name 'pyPeriodicTable' from 'rdkit.Chem'

I checked the Chem folder and found no pyPeriodicTable.py file 

I used conda install -c rdkit rdkit to install in and the installation seems OK
Could anyone provide any suggetions?

**Comment by ptosco** [comment]
Hi, there is no `pyPeriodicTable` module in the RDKit.
However, there is a `PeriodicTable` module:
http://www.rdkit.org/docs/source/rdkit.Chem.PeriodicTable.html?highlight=periodictable#module-rdkit.Chem.PeriodicTable
and
`from rdkit.Chem import PeriodicTable`
will work.

**Comment by fdmm1989** [comment]
> Hi, there is no `pyPeriodicTable` module in the RDKit.
> However, there is a `PeriodicTable` module:
> http://www.rdkit.org/docs/source/rdkit.Chem.PeriodicTable.html?highlight=periodictable#module-rdkit.Chem.PeriodicTable
> and
> `from rdkit.Chem import PeriodicTable`
> will work.

Hi ptosco,
Thanks for your reply. However, I didn't find periodictable module in my rdkit.
But I think I found where the problem is: the chemopy script has some problem in importing the periodic table. In its kappa.py script there are two statements:
from rdkit.Chem import pyPeriodicTable as PeriodicTable
periodicTable = rdchem.GetPeriodicTable()

However the periodicTable is never used, instead, PeriodicTable was used in the subsequent statements. I changed the PeriodicTable to periodicTable and everything is fine now.
